Python/bytecodes.c

```diff
@@ -4284,7 +4284,7 @@ dummy_func(
             res = PyStackRef_FromPyObjectSteal(res_o);
         }
 
-        inst(CALL_ISINSTANCE, (unused/1, unused/2, callable, self_or_null, args[oparg] -- res)) {
+        op(_CALL_ISINSTANCE, (callable, self_or_null, args[oparg] -- res)) {
```

Comment:
Are the changes in this file necessary? Each `inst` will also define an `op`.

However, a good follow-up would be guard removal. `self_or_null` is always going to be `NULL`, so in another PR you can rework the specialization to enforce this, and get rid of the guards (@diegorusso did the same thing to `CALL_LEN` in GH-133180).